woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -281,7 +281,7 @@ def test_getitem(sample_df):
     df = sample_df
     df.ww.init(
         time_index='signup_date',
-        index='id', name='dt_name',
+        index='id', name='df_name',
```

Comment:
There's a couple more uses of `dt_name` in `test_schema.py` that we should remove

Reply:
✅ 9b6d345